usecases/cluster/delegate.go

```diff
@@ -136,16 +137,19 @@ func (d *delegate) init(diskSpace func(path string) (DiskUsage, error)) error {
 	if diskSpace == nil {
 		return fmt.Errorf("function calculating disk space cannot be empty")
 	}
+	lastTime := time.Now()
+	minUpdatePeriod := time.Second + _ProtoTTL/3
 	space, err := diskSpace(d.dataPath)
 	if err != nil {
-		return fmt.Errorf("disk_space: %w", err)
```

Comment:
I'm curious in which cases we can't calculate disk space while `init` but it might work further down the line ? What is the reasoning for removing the return on error here ?

Reply:
The function might fail if d.dataPath does not exist. Instead of returning an error we log it. The delegate will retry to calculate the disk space later when the data directory exists.